src/test/scripts/functions/builtin/image_mirror_linearized.dml

```diff
@@ -0,0 +1,19 @@
+input =  read($in_file);
+
+n = nrow(input);
+m = ncol(input);
+
+input_flattened_row = matrix(input, rows=1, cols=n*m);
```

Comment:
i would like to see a test that use multiple images, not just one.

Reply:
how can I design this effectively ? since I am comparing the results of img_mirror and img_mirror_linearized it gets a bit confusing for me . I thought about linearizing the  input matrix and putting the row over and over again in a new matrix but should I do completely different matrixes ? in that case how many ?